ol/genesis-tools/src/lib.rs

```diff
@@ -9,9 +9,9 @@
 )]
 
 pub mod fetch_archive;
-pub mod fork_daemon;
+// pub mod fork_daemon;
```

Comment:
Maybe here add a comment as to why this is commented out (e.g. if the module should be kept)

Reply:
yes good point I'll document that. In this case it's a feature that won't ship, at least not in this version.